lld/ELF/Symbols.h

```diff
@@ -54,6 +54,7 @@ enum {
   NEEDS_TLSGD_TO_IE = 1 << 6,
   NEEDS_GOT_DTPREL = 1 << 7,
   NEEDS_TLSIE = 1 << 8,
+  HAS_SHARED_DEF = 1 << 9,
```

Comment:
`Symbol::flags` has a comment "Temporary flags used to communicate which symbol entries need PLT and GOT entries during postScanRelocations();". The description seems misleading with the proposed changes.

Reply:
Thanks for the comment. I added a comment to `lld/ELF/Symbols.h`